akka-docs/src/main/paradox/additional/books.md

```diff
@@ -8,23 +8,16 @@
  * [Akka in Action](https://www.lightbend.com/resources/e-book/akka-in-action), by Raymond Roestenburg and Rob Bakker, Manning Publications Co., ISBN: 9781617291012, September 2016
```

Comment:
Second edition is now here: https://www.manning.com/books/akka-in-action-second-edition